sormas-api/src/main/java/de/symeda/sormas/api/sormastosormas/SormasToSormasErrorResponse.java

```diff
@@ -16,23 +16,25 @@
 package de.symeda.sormas.api.sormastosormas;
 
 import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
 
 public class SormasToSormasErrorResponse implements Serializable {
 
-	private String message;
+	private Map<String, Map<String, List<String>>> errors;
```

Comment:
Since there is the `ValidationErrors` class this could be changed to private final `Map<String, ValidationErrors> errors;` to make it more easier to read and use. `ValidationErrors` can be named more generic if it helps in other places.

Reply:
I don't see where ValidationErrors could be reused right now, maybe later we can generalize it.